extensions/blocks/logo-gallery/logo-gallery.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Logo Gallery Block.
+ *
+ * @since 7.1.0
+ *
+ * @package Jetpack
+ */
+
+jetpack_register_block(
+	'jetpack/logo-gallery',
+	array(
+		'render_callback' => 'jetpack_logo_gallery_block_load_assets',
+	)
+);
+
+/**
+ * Logo gallery block registration/dependency declaration.
+ *
+ * @param array  $attr    Array containing the logo gallery block attributes.
+ * @param string $content String containing the logo gallery block content.
+ *
+ * @return string
+ */
+function jetpack_logo_gallery_block_load_assets( $attr, $content ) {
+	$dependencies = array();
+
+	Jetpack_Gutenberg::load_assets_as_required( 'logo-gallery', $dependencies );
```

Comment:
If you don't have need to load `view.js`, you can use `Jetpack_Gutenberg::load_styles_as_required` to avoid it.